plugins/hls-rename-plugin/src/Ide/Plugin/Rename.hs

```diff
@@ -92,8 +94,10 @@ renameProvider state pluginId (RenameParams (TextDocumentIdentifier uri) pos _pr
         -- Perform rename
         let newName = mkTcOcc $ T.unpack newNameText
             filesRefs = collectWith locToUri refs
-            getFileEdit = flip $ getSrcEdit state . replaceRefs newName
-        fileEdits <- mapM (uncurry getFileEdit) filesRefs
+            getFileEdit (uri, locations) = do
+              verTxtDocId <- lift $ getVersionedTextDoc (TextDocumentIdentifier uri)
```

Comment:
I think this may well fix the rename plugin to work properly if one of the files that we rename in has been changed in the editor!

Reply:
Yeah, I had the feeling that I am fixing a bug here.